packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -614,6 +572,10 @@ export function getValidatorApi(
       );
     }
 
+    const graffitiBuffer = toGraffitiBuffer(
```

Comment:
unrelated to this PR but could consider renaming

```suggestion
    const graffitiBytes = toGraffitiBytes(
```